chall04/vde-dios.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python3
+
+import sys
+import re
+
+
+def error(e , i):
+    file_name =  "stdin" if i == 0 else sys.argv[i]
+    if (e == 1):
+        print("%s: %s: Bad format" %(sys.argv[0], file_name))
+    elif (e == 2):
+        print("%s: %s: Can't read file" %(sys.argv[0], file_name))
+    elif (e == 3):
+        print("%s: %s: Not enough space in the given shelves" %(sys.argv[0], file_name))
```

Comment:
Using more explicit identifiers for your errors would have been better, having to go back to the function each time I see a call to guess what will be the error message is a little annoying.  
You could have used different solution:
- A global variable
- Use the `enum.Enum` class
- Create your own types of exceptions to catch them later